LICENSE

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
The license should be maintained to keep continuity with the original source

Reply:
Done